### ISSUE TITLE
Issue with URLs of the member profile pages

### ISSUE THREAD

**Issue by birdofpreyru** [appirio-tech/topcoder-app#1220]
As reported by a community member:
```
I tried to open a profile page of [[iwi]] or John Dethridge, but if I open it says "404".
John Dethridge: https://www.topcoder.com/members/John%20Dethridge
[[iwi]]: https://www.topcoder.com/members/[[iwi]]
Seems like user includes space, '[', ']', '(', ')' cannot open user page.
```

It looks to me, there are two problems: (1) username in the URL must be URL-encoded, (2) the user profile endpoint demands trailing `/`. These links work:
https://www.topcoder.com/members/John%20Dethridge/
https://www.topcoder.com/members/%5B%5Biwi%5D%5D/

The best fix will be to update the routing inside `topcoder-app`, so that it redirects to the proper profile pages even if the incoming link violates (1) and/or (2).

**Comment by ThomasKranitsas** [comment]
@birdofpreyru which branch should I use?

**Comment by birdofpreyru** [comment]
@ThomasKranitsas `dev`

**Comment by birdofpreyru** [comment]
Fixed by the commit 989654c5157ca4b4acad06e8cd4289dc4608020b
@ajefts Can I push it to prod? To do so, is it still enough just to merge the change into master, or should I also tag the change with a new version?
